src/providers/pkcs11_provider/mod.rs

```diff
@@ -269,13 +269,16 @@ impl Drop for Session<'_> {
 }
 
 /// Gets a key identifier from the Key ID Manager.
-fn get_key_id(key_triple: &KeyTriple, store_handle: &dyn ManageKeyIDs) -> Result<[u8; 4]> {
+fn get_key_id(
```

Comment:
Shouldn't this be `get_key_info`

Reply:
Yes it should! I will do the renaming of all text and filenames from key ID manager to key info manager in a subsequent PR.